docs/en/msg_docs/SensorGps.md

```diff
@@ -38,18 +38,19 @@ float32 vdop			# Vertical dilution of precision
 int32 noise_per_ms		# GPS noise per millisecond
```

Comment:
Please remove the changes under /docs 

1. The updates will come automatically from the uorb messages themselves.
2. Generally if you DID have to change these, you would only update the files under `/docs/en` - the rest are translations such as  `/docs/zh/`, and they are updated automatically by round tripping through  a translation service.